gdc_ng_models/models/audit.py

```diff
@@ -0,0 +1,28 @@
+"""Common audit metadata for our data models."""
+
+from sqlalchemy import schema
+from sqlalchemy.sql import sqltypes
+
+
+class AuditColumnsMixin:
+    """A mixin to add audit columns to a data model.
+
+    Attributes:
+        created_date: Metadata describing when a row was created.
+        updated_date: Metadata describing when a row was last updated.
+    """
+
+    def __init__(self):
+        pass
+
+    created_date = schema.Column(
```

Comment:
We have been not been consistent in how we name the equivalent fields across all of our data models, but FWIW gdcdictionary calls these `created_datetime` and `updated_datetime`. I think it would be slightly better to reuse those names, but am also fine with being slightly different.

Reply:
i like consistency.